### PR TITLE
update deploy/helm-charts/README.md

### PR DESCRIPTION
**Pull request by xinjixi**
- update README with steps to deploy API Insights with helm charts locally

### CODE REVIEW

File: deploy/helm-charts/README.md [hunk 1]
```diff
@@ -1 +1,27 @@
 # API Insights
+
+### Prerequisites
+* Install and configure the following tools:
+    * [kubernetes](https://kubernetes.io/)
+    * [kuebctl](https://kubernetes.io/docs/reference/kubectl/)
+    * [helm](https://helm.sh/)
+* Install an application for managing a local Kubernetes cluster and associated tools. For example, [Rancher Desktop](https://rancherdesktop.io/) allows you to locally set up Kubernetes, `kubectl` and `helm`. Other Kubernetes setup applications include *Minikube* and *Kind*.
+
+### Deploy API Insights using Helm
```
Comment (coliu19) [inline deploy/helm-charts/README.md:10]: How do I use the latest source code to build images used by helm? I mean how to create my customized api-insights-0.x.x.tgz?

Comment (xinjixi) [reply deploy/helm-charts/README.md:10]: Hi @coliu19, api-insights-0.x.x.tgz is for distributing packaged helm charts, here what's provided is used to deploy API Insights using source code of helm charts.

If needed, run following command to generate a package:
```
helm package ./api-insights
```

Comment (coliu19) [reply deploy/helm-charts/README.md:10]: I think there is gap between our understandings. My requirement is, I want to use all things locally from source, dont use  the existing public images. Suppose I am testing Chris's UI change locally, this fixed UI image is not published yet, I just want to test the helm with it locally. Here is the common steps:

 1. I want to checkout to some feature branch, make some changes. 
 2. Build docker images locally from source code.
 3. Use helm chart the deploy the newly created local images into my local k8s.

**Review by npateriya — APPROVED** [review]
LGTM

**Pull request merged**